### ISSUE TITLE
<html> element on docs page does not have a lang attribute

### ISSUE THREAD

**Issue by kennethbigler** [recharts/recharts#3874]
- [x] I have searched the [issues](https://github.com/recharts/recharts/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://recharts.org/en-US](https://recharts.org/en-US)

### Steps to reproduce
Go to https://recharts.org/en-US, open up lighthouse or Axe Accessibility tools, and run a scan. You will see an error for html language linking here: https://dequeuniversity.com/rules/axe/4.8/html-has-lang?application=AxeChrome

### What is expected?
language should be defined

### What is actually happening?
it is not, this can cause issues for screen reader users.

| Environment | Info |
|---|---|
| Recharts | v2.9.0 |
| React | ^18.0.0 |
| System | Mac |
| Browser | Chrome |

---
I will create a PR to fix this as well

**Comment by ckifer** [comment]
Sweet, thanks for this
